CHANGELOG.rst

```diff
@@ -6,3 +6,13 @@ v0.0.0
 ------
 
 *xxxx-xx-xx* -- Initial release.
+
+Methods/API Update
```

Comment:
A changelog is there to list big important changes. Tests are rarely mentioned in this

Reply:
Okay so should i not even mention about new methods being added. Or where should i maintain docs for these methods??

Reply:
Updated.